cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -5,44 +5,449 @@
+                                        …
+                                    </div>
+                                </div>
+                                <aside class="sub-nav_media">
+                                    <a href="/the-bureau/">
+                                        <img src="/static/img/nav-about-us-video.png" alt="About Us video image">
+                                        <p class="h4 u-mt15">
+                                          The CFPB: Working for you.
+                                        </p>
+                                    </a>
+                                    <p>
+                                        This short video covers what the CFPB is and
+                                        how we are working for American consumers.
+                                    </p>
+                                </aside>
+                            </div>
+                        </div>
+                    </li>
+                </ul>
+                [Global Eyebrow]
+            </nav>
         </div>
       codenotes:
         - |
           Structural cheat sheet:
           -----------------------
           .o-mega-menu
+            .o-mega-menu_trigger
+            .o-mega-menu_content
+                ul.o-mega-menu_content-list
+                    li.o-mega-menu_content-item
+                        .m-global-header-cta.m-global-header-cta__list
+                    li.o-mega-menu_content-item
+                        a.o-mega-menu_content-link
+                    li.o-mega-menu_content-item
+                        a.o-mega-menu_content-link
+                        .o-mega-menu_subcontent
+                            .wrapper
+                                button.o-mega-menu_subcontent-btn__back
+                                .o-mega-menu_subcontent-menu
+                                    .o-mega-menu_subcontent-title
+                                        a.o-mega-menu_subcontent-link
+                                    .o-mega-menu_subcontent-lists
+                                        ul.o-mega-menu_subcontent-list
+                                            li.o-mega-menu_subcontent-item
+                                        ul.o-mega-menu_subcontent-list
+                                            li.o-mega-menu_subcontent-item
+                                aside.sub-nav_media
   tags:
     - cfgov-organisms
 */
 
 .o-mega-menu {
 
-    .m-global-header-cta {
+    &_content-item,
+    &_subcontent-item {
+        // Override margin added by CF to list items.
+        margin-bottom: 0;
+    }
 
-        .respond-to-min( @bp-med-min, {
-            display: none;
-        } );
+    &_content {
+        &-link {
+            display: block;
+        }
     }
 
-    .m-global-eyebrow {
+    &_subcontent {
+        &-title {
+            display: block;
+            border-bottom: 1px solid @gray-50;
+        }
+
+        &-link {
```

Comment:
So my understanding of BEM is:

```
o-mega-menu                           = (B)lock
o-mega-menu_subcontent                = (E)lement of o-mega-menu block
o-mega-menu_subcontent-link           = (E)lement of o-mega-menu block
o-mega-menu_subcontent-link__disabled = (M)odifier of &_subcontent-link element in o-mega-menu block (this is a hypothetical modifier)
```

I'm not sure in BEM if you'd have an element of an element of a block, like `o-mega-menu_subcontent_link`
@Scotchester Do you have insight into this?